uwblueprint/community-fridge-kw
# fix: Unauthorized error when deleting users for admin

## Brief description. What is this change? 

### [Unauthorized error when deleting users for admin](https://www.notion.so/uwblueprintexecs/Unauthorized-error-when-deleting-users-for-admin-e76d15e71a91435d9c57b49f63fb53ae)



## Implementation description. How did you make this change?
Initial approach:
* Put back the auto-refresh code from PR #193 

Final approach:
Since no other user role (donor or volunteer) has the means to delete a user from the frontend, we just removed the `isAuthorizedByRole` check when deleting a user in `userRouter.delete`. (Since we can't figure out why bearer token is null and not set upon login and only upon refreshing as seen in the initial approach, and not sure where is the right place to set this bearer token value w/o breaking functionality of auto-logging out user after 1 hour).



## Steps to test
We need to make sure both deleting a user as admin AND auto-log out after 1 hour works.
1. Log in as admin
2. Go to User Management and delete a user.
3. User should be successfully deleted.
4. Don't log out and wait 1 hour or more
5. Go back to the page and make sure it auto-logged out for you.

## Checklist
